### PR TITLE
Feature/move to wsgi

### PR DESCRIPTION
**Pull request by aninternetof**
Changes setup.sh to use nginx

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -12,14 +12,19 @@ if [ ! -e ${ADAFRUIT_DIR} ]; then
...
+apt-get install -y python python-dev python-pip python-smbus nginx uwsgi uwsgi-plugin-python
 pip install flask
 
 pushd ${ADAFRUIT_DIR} > /dev/null
 python setup.py install
 popd > /dev/null
 
-cp www/lighttpd.conf /etc/lighttpd/lighttpd.conf
+rm -rf /var/www/rovercode > /dev/null
+mkdir /var/www/rovercode
```
Comment (cabarnes) [inline setup.sh]: I would suggest adding the `-p` option to `mkdir` so that it will work if `www` isn't there also

File: setup.sh [hunk 2]
```diff
@@ -12,14 +12,19 @@ if [ ! -e ${ADAFRUIT_DIR} ]; then
...
+ln -s $(pwd)/www /var/www/rovercode/www
+rm -f /etc/nginx/sites-enabled/rovercode > /dev/null
+ln -s $(pwd)/nginx-site /etc/nginx/sites-enabled/rovercode
```
Comment (cabarnes) [inline setup.sh]: You could add the `-f` option to `ln` to overwrite the link if it's already there and then remove the `rm` above

File: README.md [hunk 3]
```diff
@@ -15,12 +15,12 @@ rovercode runs on any single-board-computer supported by the [Adafruit Python GP
 ## Easy Setup
 First, on your robot:
 ```
-$ git clone https://git.io/rovercode.git
+$ git clone --recursive https://github.com/aninternetof/rovercode.git
+$ sudo ./setup.sh
+$ sudo ./start.sh
```
Comment (cabarnes) [inline README.md]: Do you want to mention something here about how start.sh would need to be run each power cycle but not the others?

File: www/rovercode.ini [hunk 4]
```diff
@@ -0,0 +1,11 @@
+[uwsgi]
+module = wsgi:app
+
+master = true
+processes = 5
+
+socket = /var/www/rovercode/rovercode.sock
+chmod-socket = 777
```
Comment (cabarnes) [inline www/rovercode.ini]: I'm not sure that it needs the execute permission and do you want it to be accessed by anyone?

**Comment by aninternetof** [comment]
Ok, comments incorporated. Lemme know if I can pull!

**Pull request merged**